libr/core/core.c

```diff
@@ -1303,15 +1298,14 @@ static bool find_autocomplete(RLine *line) {
 			}
 			// fallback to command listing
 		}
-		int length = strlen (arg);
+		int length = strlen (arg), j = 0;
 		for (i = 0; j < (TMP_ARGV_SZ - 1) && i < parent->n_subcmds; i++) {
 			if (!strncmp (arg, parent->subcmds[i]->cmd, length)) {
 				tmp_argv[j++] = parent->subcmds[i]->cmd;
 			}
 		}
 		tmp_argv[j] = NULL;
 		line->completion.argc = j;
-		line->completion.argv = tmp_argv;
```

Comment:
why are we computing `tmp_argv` if we don't set it?

Reply:
It has been set before `	line->completion.argv = tmp_argv;`

Reply:
Where exactly? I cannot find it.